crates/wast/src/resolve/deinline_import_export.rs

```diff
@@ -1,16 +1,16 @@
 use crate::ast::*;
-use crate::resolve::gensym;
+use crate::resolve::gensym::Gensym;
 use std::mem;
 
-pub fn run(fields: &mut Vec<ModuleField>) {
+pub(crate) fn run(fields: &mut Vec<ModuleField>, gensym: &mut Gensym) {
```

Comment:
How come the gensym bits went from something global to a struct-to-pass-around?

(IIRC I originally passed it around but it was such a pain to do so I stopped doing that)

Reply:
That was just me trying to understand name resolution for nested modules and wanting to understand what state was kept where. I can reinstate the thread-local if it's clearer that way, but the current code doesn't need to pass it around that many places.